src/user.js

```diff
@@ -24,10 +24,43 @@ class User {
     this.recipesToCook.splice(i, 1);
   }
 
-  // updatePantry() {
-  // //remove ingredient quantities from user Pantry, leave till later [ ]
-  // //remove recipe from array [x]
-  // }
+  removePantryIngridients(recipe) {
+    recipe.ingredients.forEach((item) => {
+    fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData', {
+      method: 'POST',
+      headers: {
+  	     'Content-Type': 'application/json'
+       },
+       body: JSON.stringify({
+         "userID": this.id,
+         "ingredientID": item.id,
+         "ingredientModification": -(item.quantity.amount)
+       })
+     })
+     .then(response => response.json())
+     .then(respone => console.log(response))
+     .catch(error => console.log(error));
+   });
+  }
+
+  addNeededPantryIngridients(recipe) {
+    this.checkHowMuchMore(recipe).forEach((item) => {
+    fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData', {
+      method: 'POST',
+      headers: {
+  	     'Content-Type': 'application/json'
+       },
+       body: JSON.stringify({
+         "userID": this.id,
+         "ingredientID": item.id,
+         "ingredientModification": item.quantityNeeded
+       })
+     })
+     .then(response => response.json())
+     .then(response => console.log(response))
+     .catch(error => console.log(error));
+   });
+  }
```

Comment:
I'm curious how this implementation of the fetches within the class will affect our ability to test it.

Reply:
That's a great question I wish I had an answer for, I had vascilated between options, and it seemed like the project spec hinted more to having this functionality on the class object and less on the DOM object.

Reply:
That works for me.